### PR TITLE
fix restart command

### PR DESCRIPTION
**Pull request by DaStormer**
- `shardIDForGuildID()` is a static method, so it needs to be called directly on the class constructor
- Defined the guildID parameter as `args.server` instead of `msg.guild.id`

### CODE REVIEW

File: src/commands/rover/RestartCommand.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 const Accolades = require('../../Accolades.json')
 const Command = require('../Command')
+const Discord = require('discord.js')
```
Comment (Wolftallemo) [inline src/commands/rover/RestartCommand.js:3]:
```suggestion
const { ShardClientUtil } = require('discord.js')
```

I would prefer object destructuring over importing everything here.

**Comment by Wolftallemo** [comment]
But anyways, it's ok otherwise.

And I won't be writing code at 4am ever again

File: src/commands/rover/RestartCommand.js [hunk 2]
```diff
@@ -30,9 +31,8 @@ class RestartCommand extends Command {
       await msg.reply('Restarting...')
       process.exit()
     }
-    const shard = msg.client.shard.shardIDForGuildID(msg.guild.id, msg.client.shard.count).catch(() => {
-      msg.reply('An invalid server id was given!')
-    })
+    if(isNaN(args.server)) return msg.reply("An invalid server id was given!")
+    const shard = Discord.ShardClientUtil.shardIDForGuildID(args.server, msg.client.shard.count)
```
Comment (Wolftallemo) [inline src/commands/rover/RestartCommand.js:35]:
```suggestion
    const shard = ShardClientUtil.shardIDForGuildID(args.server, msg.client.shard.count)
```

I forgot this part 🤦‍♀️

**Review by Wolftallemo — APPROVED** [review]
Looks good

**Comment by Rythian2277** [comment]
....??? Was this tested sufficiently before pulling?

**Comment by DaStormer** [comment]
> ....??? Was this tested sufficiently before pulling?

I tested it before creating the PR; works as expected. The previous code was returning this error: 
img

**Comment by Rythian2277** [comment]
Ah, alright, I will have to make some adjustments anyway as there is intended functionality that you overwrote.

**Comment by Rythian2277** [comment]
Ah, alright, I just needed to make sure as sometimes Discord.js acts unexpectedly.

**Pull request merged**